<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Send_message</class>
 <widget class="QDialog" name="Send_message">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Send</string>
   </property>
   <widget class="QPushButton" name="pushButton_ok_send">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_send">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_Send_message_to">
       <property name="text">
        <string>Message to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_dst"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Message:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_body"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
